docs/source/features.rst

```diff
@@ -169,7 +169,7 @@ to fine-tune instance lifetimes.
 
 Since files can hold on to write caches and read buffers,
 the instance cache may cause excessive memory usage in some situations; but normally, files
-will get ``close``d, and the data discarded. Only when there is also an unfinalised transaction or
+will get ``close``\d, and the data discarded. Only when there is also an unfinalised transaction or
```

Comment:
This construct is a little weird, feel free to reword to avoid the extra syntax

Reply:
Reworded.